src/main/scala/org/codeoverflow/chatoverflow/requirement/service/rcon/impl/RconInputImpl.scala

```diff
@@ -0,0 +1,19 @@
+package org.codeoverflow.chatoverflow.requirement.service.rcon.impl
+
+import org.codeoverflow.chatoverflow.WithLogger
+import org.codeoverflow.chatoverflow.api.io.input.RconInput
+import org.codeoverflow.chatoverflow.registry.Impl
+import org.codeoverflow.chatoverflow.requirement.InputImpl
+import org.codeoverflow.chatoverflow.requirement.service.rcon.RconConnector
+
+@Impl(impl = classOf[RconInput], connector = classOf[RconConnector])
+class RconInputImpl extends InputImpl[RconConnector] with RconInput with WithLogger {
+  override def getCommandOutput(command: String): String = sourceConnector.get.sendCommand(command)
+
+  /**
+    * Start the input, called after source connector did init
+    *
+    * @return true if starting the input was successful, false if some problems occurred
+    */
+  override def start(): Boolean = sourceConnector.get.isLoggedIn
+}
```

Comment:
Why is there no `stop()` method implemented here? That way it wouldn't even compile.  

I wonder how this can happen... 😕  
Was it lost during commit or haven't you properly tested your implementation?

Reply:
I was using a little bit older version of chatoverflow (21 days ago) which does not have the stop method.
I will fix this

Reply:
Ahh that explains a lot :sweat_smile: